### PR TITLE
Feature: ONT metadata updates

### PR DESCRIPTION
**Pull request by marcomoscasgr**
This PR will add:
- `update-ont-metadata` script to run ONT metadata updates within a time window until the current date (since 14 days before by default)
- a logging configuration file example `logging_syslog.conf` built to log ERROR level messages to syslog
- a test in `test_ont.py` that calls the script with `subprocess` and essential options

### CODE REVIEW

**Comment by marcomoscasgr** [comment]
I added `tests/data/logging_syslog.conf` as an example of how to set up a log dispatcher for syslog in python, it is not used by the tests. Should I keep it in the repository?

**Comment by kjsanger** [comment]
> I added `tests/data/logging_syslog.conf` as an example of how to set up a log dispatcher for syslog in python, it is not used by the tests. Should I keep it in the repository?

It shouldn't be under tests is the tests don't use it. However, there should be documentation so that people know how to create a log config, so it's still useful as an example.

**Review by kjsanger — CHANGES_REQUESTED** [review]

File: scripts/update-ont-metadata [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+)
+parser.add_argument(
+    "--dbconfig",
+    help="Configuration file for database connection.",
+    type=argparse.FileType("r"),
+    required=True,
+)
+parser.add_argument(
+    "--logconf", help="A logging configuration file.", type=argparse.FileType("r")
+)
```
[the remark this answers is not in the document]
Comment (marcomoscasgr) [reply scripts/update-ont-metadata:66]: Yes, I better choose always the long-form with no shortname

File: tests/test_ont.py [hunk 2]
```diff
@@ -182,3 +183,32 @@ def test_find_updates_for_experiment_slot(self, ont_synthetic, mlwh_session):
             )
         ]
         assert updated == expected_colls, "Found slot 1 from simple experiment 1"
+
```
[the remark this answers is not in the document]
Comment (marcomoscasgr) [reply tests/test_ont.py:186]: To test the script, Should I test only the return code of subprocess?

[the remark this answers is not in the document]
Comment (kjsanger) [reply tests/test_ont.py:186]: The script essentially contain a single method call to a tested method. I don't think we gain anything by maintaining tests for them.

[the remark this answers is not in the document]
Comment (marcomoscasgr) [reply tests/test_ont.py:186]: ok, I will remove the test at this point.

File: scripts/update-ont-metadata [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    db_port = mysql_section.get("port")
+    db_name = mysql_section.get("schema")
+
+    connection_uri = f"mysql+pymysql://{db_user}:{db_pwd}@{db_host}:{db_port}/{db_name}"
+
+    engine = sqlalchemy.create_engine(connection_uri)
+    with Session(engine) as session:
+        mu = MetadataUpdate()
+        updated = []
+        if args.begin_date:
```
[the remark this answers is not in the document]
Comment (marcomoscasgr) [reply scripts/update-ont-metadata]: I added a default `datetime` instance directly in the `add_argument` method of `argparse`. That should be a clean way to do it.

**Pull request merged**